lib/sha256/hash.js

```diff
@@ -0,0 +1,81 @@
+var Buffer = require('safe-buffer').Buffer
```

Comment:
Where is this file from?

Reply:
This file is from the sha.js package, license from this it's referenced on the readme.md file. Should add a link to the original inside this file?

Reply:
consider removing safe buffer and use Buffer.alloc and Buffer.from (built right into javascript)

Reply:
> consider removing safe buffer and use Buffer.alloc and Buffer.from (built right into javascript)
> refer here: [safe-buffer](https://www.npmjs.com/package/safe-buffer) on npmjs

If it's to reduce the size even more then I don't think safe-buffer is the place too look at. The library is small from the looks of it, and if I want it out of the compiled file I need to remove all references to it otherwise it will still get loaded.